packages/manager/modules/vrack/src/dashboard/vrack.html

```diff
@@ -203,6 +203,12 @@ <h3 data-translate="vrack_introduction_title"></h3>
                     >
                     </ng-include>
                 </div>
+                <vrack-add-ipv6-modal
+                    data-is-open-modal="$ctrl.addIpv6Modal"
+                    data-on-confirm="$ctrl.addServiceAction()"
+                    data-on-cancel="$ctrl.resetAction()"
+                    data-state="$ctrl.addIpv6ModalState"
+                ></vrack-add-ipv6-modal>
```

Comment:
**nitpick:** Add a new angularjs state for modal and just do the state naviagation from the dashboard.